demaceo/whats-cookin
# DOM interactions, Styling additions, Class definitions

##### Note: Working with Demaceo via Atom teletype

### What's this PR do?
- Implementation of:
1. User list
2. Profile menu
3. User view (pantry, favorites, to cook)
4. Log in/Log out 
5. Links class files for DOM interaction
6. User name and section displayed in navbar upon login
7. clicking into/out of search bar expands/contracts search bar
8. Recipes and users randomly generate for 'others cookin' section' and staff picks section
9. New image assets
10. Functional hamburger menu with animation

- Fixes:
1. navbar styling no longer broken with addition of user select option
2. logging in only occurs when clicking on a user name from the user dropdown menu

### How should this be manually tested?

- On landing, user profile elements should not be accessible, you should be able to pick a user ('login') from the users icon in the navbar
- Can login from any view
- Login will access user menu (click to drop down)
- Access any of the sections - user name and section should appear in navbar
- Should be accessed from any view
- Clicking logout will reveal homepage, hide profile options, and reveal the choose user dropdown again
- Should not be able to access user sections when not logged into a user
-Hamburger menu should slide open on click of hamburger icon
- Close menu by clicking 'X' in right corner
- Hamburger links are not currently functional
